platform-common-date/src/main/java/com/softicar/platform/common/date/Day.java

```diff
@@ -214,6 +215,30 @@ public static Day fromYMD(int year, int month, int day) {
 		return Day.get(absoluteDay);
 	}
 
+	/**
+	 * Returns the {@link Day} for the given triplet of year, month and day of
+	 * the month.
+	 *
+	 * @param year
+	 *            the absolute number of the year
+	 * @param month
+	 *            the number of the month [1,12]
+	 * @param day
+	 *            the number of the day [1,n], where <i>n</i> is the number of
+	 *            days in the month
```

Comment:
```suggestion
	 *            days in the month of the respective year
```
...to hint to leap years.